gpio_controllers/test/test_gpio_command_controller.cpp

```diff
@@ -489,8 +571,13 @@ TEST_F(
      {"command_interfaces.gpio2.interfaces", std::vector<std::string>{"ana.1"}},
      {"state_interfaces.gpio1.interfaces", std::vector<std::string>{"dig.1", "dig.2"}},
      {"state_interfaces.gpio2.interfaces", std::vector<std::string>{"ana.1"}}});
-  move_to_activate_state(
-    controller_->init("test_gpio_command_controller", "", 0, "", node_options));
+  controller_interface::ControllerInterfaceParams params;
+  params.controller_name = "test_gpio_command_controller";
+  params.robot_description = "";
+  params.update_rate = 0;
+  params.node_namespace = "";
+  params.node_options = node_options;
+  move_to_activate_state(controller_->init(params));
```

Comment:
could these params be made part of the text fixture please? this is a lot of copy-paste of the same code in this file